src/modules/createGame.js

```diff
@@ -0,0 +1,24 @@
+export const setID = (object) => {
+  window.localStorage.setItem('id', object.result);
+};
+
+export const getID = () => {
+  const string = window.localStorage.getItem('id');
+  return string.slice(14, 35);
+};
+
+const createGame = async (data) => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    body: data,
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
+  response.json().then((object) => {
+    setID(object);
+    console.log(getID());
```

Comment:
[REQUIRED] Please do not forget to remove `console.log` statement from the codebase once your check is done, it is important in terms of `clean code` that applies to [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).